### PR TITLE
Follow up to pull request #11

### PR DESCRIPTION
**Pull request by robmen**

### CODE REVIEW

**Comment by 2dave** [comment]
When reviewing a pull request and then merging it for another developer is it typical to comment on each commit?

**Review by 2dave — COMMENTED** [review]
Yep - that is what I was asking about when I was talking about another variable that you just assign the length to. It does make it easier to read.

**Review by 2dave — COMMENTED** [review]
I had something like that originally. I was finding a lot of different 'best practices' online. Many sites recommended just writing to a log and not outputting to the console.

**Review by 2dave — COMMENTED** [review]
This is used a lot in beginner level material when teaching about file streams. I need to try to understand this topic a little more. If the program gets a general exception won't the program still crash?  or because the code is in a try block it won't? or perhaps it should crash if you get an exception outside of file not found.

**Comment by robmen** [comment]
Hmm, this is really hard to follow. To leave a comment on a particular piece of code, click on the commit, float over the changed lines and you'll see a blue plus sign on the left. Click that and you can leave a comment specifically on a line so there is context about your comments.

It's hard to know what you are referring to with the comments above because there is no context to them.

**Comment by 2dave** [comment]
Sorry about that! That was first time I commented on a pull request. I see the blue plus sign you are talking about and will do that next time.
